nossas/bonde
# Hotfix spanish translate

Fix spanish translation to recipient settings of community.

## How to test

1. Go to recipient page in settings of community` and check if your page is translated to spanish.

** Use a tool to translate page or change it language in your browser
